crates/solver/src/settlement_submission/submitter/custom_nodes_api.rs

```diff
@@ -52,33 +52,30 @@ impl TransactionSubmitting for CustomNodesApi {
             .collect::<Vec<_>>();
 
         loop {
-            let (result, _index, rest) = futures::future::select_all(futures).await;
+            let (result, index, rest) = futures::future::select_all(futures).await;
+            let lable = format!("custom_nodes_{}", index);
```

Comment:
ubernit: Edition 2021 niceness!
```suggestion
            let lable = format!("custom_nodes_{index}");
```